Framework/MPIAlgorithms/test/BroadcastWorkspaceTest.h

```diff
@@ -57,10 +57,18 @@ class BroadcastWorkspaceTest : public CxxTest::TestSuite {
     API::MatrixWorkspace_const_sptr outWS =
         broadcaster.getProperty("OutputWorkspace");
     TS_ASSERT_EQUALS(inWS->size(), outWS->size());
+
+    const auto &inX = inWS->x(0);
```

Comment:
You didn't really need to do this. `inWS->x(0)[i]` would have been sufficient. You do not have to change it but it seems like a lot of unecessary extra code. Actually if you include `#include <MantidTestHelpers/HistogramDataTestHelpers.h`, header file name might not be correct, you can directly compare the two using `TS_ASSERT_EQUALS(inWS->x(0), outWS->x(0));`